packages/atlas/lib/src/provider.dart

```diff
@@ -29,6 +29,7 @@ abstract class Provider {
     final ArgumentCallback<AtlasController> onMapCreated,
     final ArgumentCallback<CameraPosition> onCameraPositionChanged,
     final ArgumentCallback<LatLng> onLocationChanged,
+    final ArgumentCallback<bool> onPan,
```

Comment:
Why is this an ArgumentCallback<bool>?

Reply:
I don't need it tbh. I will replace it for a `VoidCallback`, what do you think?